gen_docs

```diff
@@ -36,4 +36,4 @@ bash "$source"scripts/mk_all.sh
 lean src/entrypoint.lean >export.json
 bash "$source"scripts/rm_all.sh
 
-python3 print_docs.py -r "$source" -w "$site_root" "${args[@]}"
+lean-doc-gen -r "$source" -w "$site_root" "${args[@]}"
```

Comment:
Please make it possible to run doc-gen and gen_docs without installing the package.  `python -m lean_doc_gen` fails here.

Reply:
Why do we care about supporting non-installed packages?

Note that even with the package installed, `python -m lean_doc_gen` fails here; but that's only because it picks up the non-installed version in the current directory. If I moved `lean_doc_gen` to `python_src/lean_doc_gen`, or just changed the working directory when running this script, then that wouldn't happen.